### DISCUSSION TITLE
Cursor jumps to first target in operator-pending mode for char movements

### DISCUSSION THREAD

**Discussion by TxHawks** [folke/flash.nvim#186] [Q&A]
When I use char-mode movements in operator-pending mode, e.g., `df<char>`, the cursor will jump to the first instance of `<char>` and will then carry out the operator-pending action - in this case, delete.

So if I have my cursor at the beginning of the following line:
```
A random line of text
```

and then hit `dte`, the cursor will to the `n` in `line`, and delete from there to the label I pick or the first one if I just hit `enter`.

Is this a bug or do I have something misconfigured?

This is my config:
```lua
{
  incremental = true,
  jump = {
    -- Disabling this was the first thing I tried. Didn't help
    autojump = false,
  },
  label = {
    current = false,
  },
  modes = {
    char = {
      jump_labels = true,
      label = { exclude = "abcdehijklrxw" },
    },
  },
  search = {
    incremental = true,
  },
},
```

**Comment by dedebenui** [comment]
for `d`, you cat repeat the last operation with `.`. In your example, `dte.` would leave you with the text `ext`. You would only have `e of ` in your register though. For other operations like `c`, `y` or `d` where you want to paste what you delete later, I would suggest to go in visual mode first and select what you want there: `vs<find your spot>c`, `vs<find your spot>d`, ...
